doc/source/user_guide/classical_uses/lez_regionalization.rst

```diff
@@ -269,8 +269,8 @@ Since we have returned the optimized neural network, we can visualize what it co
 
     opt_ann.net
 
-The information displayed tells us that the default neural network is composed of 2 hidden dense layers followed by ``ReLU`` activation functions 
-and a final layer followed by a ``Sigmoid`` function. To scale the network output to the boundary condition, a ``MinMaxScale`` function is applied. 
+The above information indicates that the default neural network is composed of 3 hidden dense layers, each followed by a ``ReLU`` activation function.
+The output layer is followed by a ``TanH`` function. To scale the network output to the boundary condition, a ``MinMaxScale`` function is applied.
```

Comment:
The output layer is followed by a "tanh" hyperbolic tangent function and it outputs in \left]-1;1\right[ are scaled  to given conceptual parameter bounds using a ``MinMaxScale`` function.

Reply:
Exactly !